build.sbt

```diff
@@ -89,7 +88,7 @@ val embeddedPostgres = "com.opentable.components" % "otj-pg-embedded" % "0.13.4"
 val dbTestingStack = Seq(embeddedPostgres)
 
 val catsEffectStack = Seq(
-  "org.typelevel" %% "cats-effect-laws" % "2.3.3" % Test
+  "org.typelevel" %% "cats-effect-laws" % "3.1.1" % Test
```

Comment:
I think we have to stick with cats-effect 2, as that's what's tapir is currently using